CHANGELOG

```diff
@@ -1,3 +1,7 @@
+Version 1.3.3 (2022-09-13)
+--------------------------
+Add `experimental` key (#112)
```

Comment:
nit - issue title is different (maybe change the issue title too)?

Reply:
Issue title changed to match